### PR TITLE
fix(typography): add Zilla as well as type scale and relevant scss to minimalist

### LINKED ISSUE #47: Typography and typescale
Our typography is not yet defined in Minimalist. We also need a type scale.

### PR DESCRIPTION
**Pull request by schalkneethling**
Add Zilla as well as type scale and relevant `scss` to minimalist

@mindy r?

fix #47

### CODE REVIEW

File: pages/css/mdn-minimalist.css [hunk 1]
```diff
@@ -62,59 +62,39 @@ kuma/static/styles/components/compat-tables/_vars.scss
...
-*:before,
-*:after {
-  box-sizing: inherit; }
+/* Fonts */
+/* Based on a modular scale with a base of 18px and a 
+   perfect fourth ratio of 1.333 - 
+   https://type-scale.com/?size=18&scale=1.333&text=A%20Visual%20Type%20Scale&font=Zilla%20Slab&fontweight=700&bodyfont=Fira%20Sans&bodyfontweight=400&lineheight=1.65&backgroundcolor=white&fontcolor=%23333&preview=false */
+@font-face {
+  font-family: zillaslab;
+  font-display: swap;
```
Comment (mindy) [inline pages/css/mdn-minimalist.css:71]: Didn't know about this property-- very neat!

Comment (schalkneethling) [reply pages/css/mdn-minimalist.css:71]: It essentially boils down to how important it is that the font family does eventually load. For us it is rather important that it loads but, we do not want to display invisible text while the custom font is loaded, so we use `swap`. This essentially says, use a fallback, and as soon as the custom font is available, swap it in.

More details around the other options are nicely written up here:
https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display#The_font_display_timeline

**Comment by schalkneethling** [comment]
Hey @mindy, thanks for taking a look at the PR. I did not see any concern raised so, in order to move forward with https://github.com/mdn/mdn-fiori/issues/147 I am going to go ahead and merge.

If there is anything you would like to add, remove or change with regards to this, please let me know. This is all open for discussion. Thank you.

**Comment by mindy** [comment]
Thanks @schalkneethling! Sorry, I didn't know it was blocking-- yes, this all looks good to me. Thank you for merging it in.

**Pull request merged**